rlkit/torch/td3/td3.py

```diff
@@ -99,6 +104,14 @@ def _do_training(self):
         target_q1_values = self.target_qf1(next_obs, noisy_next_actions)
         target_q2_values = self.target_qf2(next_obs, noisy_next_actions)
         target_q_values = torch.min(target_q1_values, target_q2_values)
+
+        if self.clip_q:
+            target_q_values = torch.clamp(
+                target_q_values,
+                -1/(1-self.discount),
```

Comment:
Can you make this a parameter rather than hard-coding it? It could be something like:

```
if max_q_value is None:
  max_q_value = -1/(1-self.discount)   # for HER sparse rewards.
```
in `__init__`.